README.md

```diff
@@ -9,9 +9,7 @@ In order to use the Findologic service you need to install:
 See also [OXID documentation](https://docs.oxid-esales.com/developer/en/6.2/development/modules_components_themes/module/tutorials/module_setup.html).
```

Comment:
As discussed, please remove the text in the Installation section and just link to https://docs.findologic.com/doku.php?id=integration_documentation:plugin:en:direct_integration:oxid